### PR TITLE
Array methods

### PR DESCRIPTION
**Pull request by eric-k-chu**
sss

### CODE REVIEW

File: array-methods/map.js [hunk 1]
```diff
@@ -0,0 +1,11 @@
+const prices = [42.42, 10, 28.2234, 3.2, 5, 12];
+
+const priceObjs = prices.map((element) => ({
+  price: element,
+  salePrice: element / 2,
+}));
+
+const formattedPrices = prices.map((element) => `$${element}`);
```
Comment (shawnkost) [inline array-methods/map.js:8]: For this one, you should only see 2 digits after the decimal point

**Review by shawnkost — CHANGES_REQUESTED** [review]
One change here Eric!

**Comment by eric-k-chu** [comment]
ss

**Review by shawnkost — APPROVED** [review]
👍🏻

**Pull request merged**